src/internal/queries/getEventsByCreationNumber.graphql

```diff
@@ -0,0 +1,12 @@
+query getEventsByCreationNumber($address: String, $creationNumber: bigint) {
```

Comment:
you are going to do getEventsByEventHandle and getEvents, so prob want to switch this to passing in a where condition - same comment @0xmaayan made in https://github.com/aptos-labs/aptos-ts-sdk/pull/15#discussion_r1356388435

Reply:
That make sense!